z03-data-binding.md

```diff
@@ -309,7 +308,7 @@ nextrects.enter().append('rect'); // adds one element
 
 ## Transitions `selection.transition()`
 
-The key function also important in case parts of our objects change -- if we
+The key function <isindex></isindex> also important in case parts of our objects change -- if we
```

Comment:
What's the `<isindex>` tag? Is this supposed to be: `function is also important` ?